### PR TITLE
QRadar: Fix pagination

### PR DESCRIPTION
**Pull request by omercnet**
Pagination didn't properly go over all the outstanding incidents

### CODE REVIEW

File: Integrations/integration-QRadar.yml [hunk 1]
```diff
@@ -235,13 +235,13 @@ script:
             var totalOffenses = res.length;
             var lastCallOffenses = res.length;
             while (startTime && lastCallOffenses >= offensesPerCall) {
-                var min = res.length;
-                var max = min + offensesPerCall;
+                var from = totalOffenses + 1;
+                var to = min + offensesPerCall;
```
Comment (hirshbergshachar) [inline Integrations/integration-QRadar.yml]: @omercnet does the new code work? it seems in this line that we are calling the variable `min`, but it doesn't exist anymore...

Comment (omercnet) [reply Integrations/integration-QRadar.yml]: thanks! fixed

**Comment by anara123** [comment]
@omercnet we can do kind of mock inside the code itself. lets sit on it together

**Comment by omercnet** [comment]
this isn't the latest fix - waiting for customer to test before pushing here

**Pull request merged**